### PR TITLE
typo fix

### PR DESCRIPTION
**Pull request by vrkansagara**
|    Q          |   A
|-------------- | ------
| Documentation | yes
| Bugfix        | no
| BC Break      | no
| New Feature   | no
| RFC           | no
| QA            | no

### Description
typo fix

### CODE REVIEW

File: docs/book/event-manager.md [hunk 1]
```diff
@@ -53,13 +53,13 @@ Handled event "do", with parameters {"foo":"bar","baz":"bat"}
...
 > callback can be attached as a listeners: PHP function names, static class
-> methods, object instance methods, functors, or closures. We use closures
+> methods, object instance methods, function, or closures. We use closures
 > within this post for illustration only.
 
 ### Event instances
 
 `trigger()` is useful as it will create a `Laminas\EventManager\Event` instance for
-you. You may want to create such an instance manually; for instance, you may
+you. You may want to create such an instance manually for instance, you may
```
Comment (froschdesign) [inline docs/book/event-manager.md]: Thank you for your time and this contribution! 👍 

The whole sentence is difficult to read and understanding the content is therefore also difficult. There are too many repetitions of the same words. A list of the different possibilities would improve readability.

Comment (vrkansagara) [reply docs/book/event-manager.md]: So let me revert with the original one.

Comment (froschdesign) [reply docs/book/event-manager.md]: I'm sorry, I think my wording was inadequate. With "a list of the different possibilities" I mean an enumeration / bullet list.

**Comment by michalbundyra** [comment]
Thanks, @vrkansagara!

**Pull request merged**